app/models/course/settings/assessments_component.rb

```diff
@@ -27,10 +27,16 @@ def email_enabled?(category, key)
     def valid_email_setting_key?(key)
       category_email_setting_items.key?(key)
     end
+
+    # Each item here is 1 tab under the category
+    def category_lesson_plan_item_settings(category)
```

Comment:
I think we can put this as an instance method instead of putting it as a class method and delegating. `email_enabled?` (and hence `category_email_setting_items`) is a class method because it is used when sending email, which is async and out of the typical request cycle.